content/docs/fundamentals/converting/resolving-errors@ko.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: 유효성 검사 오류 해결
+$order: 2
+toc: true
+---
+
+[TOC]
+
+이 섹션에서는 AMP 페이지에 있는 AMP 유효성 검사 오류 및 해결 방법을 살펴봅니다.  콘솔에 따라 오류가 아래와 다른 순서로 나타날 수 있습니다.
+
+## 문자 집합 포함
+
+우선 다음 오류를 해결해 보겠습니다.
+
+<pre class="error-text">
+필수 'meta charset=utf-8' 태그가 누락되었거나 잘못되었습니다.
+</pre>
+
+AMP에서 텍스트를 올바르게 표시하려면 페이지의 문자 집합을 지정해야 합니다. 또한 메타 문자 집합 정보는 `<head> ` 태그의 첫 번째 하위 요소여야 합니다. 이 태그를 첫 번째로 사용해야 하는 이유는 메타 문자 집합 태그 앞에 추가된 콘텐츠를 다시 해석하는 일을 피하기 위해서입니다.
+
+다음 코드를 `<head>` 태그의 첫 번째 줄에 **추가**합니다.
+
+
+```html
+<meta charset="utf-8" />
+```
+
+파일을 **저장**하고 페이지를 새로고침합니다. 문자 집합 오류가 더 이상 표시되지 않는지 확인합니다.
+
+## 표준 링크 포함
+
+이제 다음 오류를 살펴보겠습니다.
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).